schema-engine/connectors/sql-schema-connector/src/warnings/generators.rs

```diff
@@ -564,3 +604,23 @@ pub(super) fn row_level_security_tables_found(affected: &[Model]) -> Warning {
         affected: serde_json::to_value(affected).unwrap(),
     }
 }
+
+pub(crate) fn check_constraints_found(affected: &[CheckConstraint]) -> Warning {
+    let message = "These tables have check constraints, which are not yet fully supported. Read more: https://pris.ly/d/postgres-check-constraints";
+
+    Warning {
+        code: 31,
+        message: message.into(),
+        affected: serde_json::to_value(affected).unwrap(),
+    }
+}
+
+pub(crate) fn exclusion_constraints_found(affected: &[ExclusionConstraint]) -> Warning {
+    let message = "These tables have exclusion constraints, which are not yet fully supported. Read more: https://pris.ly/d/postgres-exclusion-constraints";
```

Comment:
Are these messages usually in plural even when triggered for one?

Reply:
Yes, as `affected` is an array, we don't usually check whether such array has 1 element or more.
We only know that it is definitely not empty.

Example:

```
        [
          {
            "code": 32,
            "message": "These are exclusion constraints, which are not yet fully supported. Read more: https://pris.ly/d/postgres-exclusion-constraints",
            "affected": [
              {
                "name": "room_reservation_room_id_tstzrange_excl",
                "definition": "EXCLUDE USING gist (room_id WITH =, tstzrange(reserved_at, reserved_until) WITH &&) WHERE ((NOT canceled))"
              }
            ]
          }
        ]
``` 

Note: I have noticed that the original warning message didn't make full sense (I originally planned on including the table referenced by the constraints, but I had troubles retrieving it an is not that relevant / useful for the stopgap)